conn.go

```diff
@@ -3,9 +3,54 @@
 package tftp
 
 import "net"
+import "fmt"
+import "strconv"
 
 const udp = "udp"
 
+var localhost string = determineLocalhost()
```

Comment:
`determineLocalhost` called not only at tests start, also at the start of any process that uses the package. It is little bit of a hassle to fix: Probably function and variable should be moved to `_test` (e.g. `tftp_test.go`) file, but in order to do that it is necessary to move `localSystem` there too, etc...

It could be done similar to https://github.com/pin/tftp/pull/26 to completely get rid of `conn.go` and `conn_darwin.go`. It should be OK, with this change _darwin workaround is not required anymore.